evaluation/run_evaluation.py

```diff
@@ -1,39 +1,44 @@
 import os
-
+import pickle
 from src.collection_utils.evaluate_collection import (
     assess_retrieval_accuracy,
     get_data_for_evaluation,
-    get_all_regex_ids,
     assess_scroll_retrieval,
 )
 from src.utils.utils import load_qdrant_client, load_config
+from dotenv import load_dotenv
+
+load_dotenv()
 
 # Get env vars
 PUBLISHING_PROJECT_ID = os.getenv("PUBLISHING_PROJECT_ID")
 EVALUATION_TABLE = os.getenv("EVALUATION_TABLE")
+EVALUATION_TABLE = f"`{EVALUATION_TABLE}`"
 QDRANT_HOST = os.getenv("QDRANT_HOST")
 QDRANT_PORT = os.getenv("QDRANT_PORT")
-EVAL_COLLECTION_NAME = os.getenv("EVAL_COLLECTION_NAME")
+EVAL_COLLECTION_NAME = os.getenv("COLLECTION_NAME")
```

Comment:
Should this env var be "EVAL_COLLECTION_NAME", i.e. the name of the eval collection in env vars? Otherwise COLLECTION_NAME could be our prod collection